src/shared/components/homepage/header/index.tsx

```diff
@@ -1,30 +1,32 @@
 import Link from "next/link";
 import Image from "next/image";
-import Logo from "@/assets/survivor-tribe-logo.png";
-import Navbar from "../../navbar";
+import HeroImg from "@/assets/jeff_probst.png";
+/* import Navbar from "../../navbar"; */
 import { Button } from "../../ui/button";
-import VisitorCounter from "../../visitor-counter/VisitorCounter";
+import TopNavBar from "../../top-sticky-navbar";
```

Comment:
Can you utilize alias paths?

```suggestion
import TopNavBar from "@/components/...
```